mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from mev_inspect.schemas.classified_traces import (
+    Classification,
+    ClassifiedTrace,
+    # CallTrace,
+    DecodedCallTrace,
+)
+from mev_inspect.schemas.liquidations import Liquidation
+
+
+# Inspect list of classified traces and identify liquidation
+def get_liquidations(traces: List[ClassifiedTrace]):
+    tx = []
+    liquidations = []
+    result = []
+
+    # Protocol contract address must be in included, below is AaveLendingPoolCoreV1
+    addrs = [
+        "0x3dfd23A6c5E8BbcFc9581d2E864a68feb6a076d3",
+        "0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9",
+        "0x398eC7346DcD622eDc5ae82352F02bE94C62d119",
+    ]
+
+    # Used to remove double-counted 'from' transfers
+    from_doubles = []
+    transfers_to = []
+    transfers_from = []
+
+    # For each trace
+    for trace in traces:
+
+        if isinstance(trace, DecodedCallTrace):
+
+            # Check for liquidation and register trace and unique liquidator
```

Comment:
rather than having this be a comment, better to break up the code so it reads the same as the comment would - break this inner piece out to a named function

Reply:
Don't understand the purpose of this one since this is just the execution of the program and some core logic. Passing a function here will make it much more difficult to append to the lists which are all defined locally in the function, so will add more work to pass these as args to a new function

Reply:
this was the case with the others but the increment was reasonable; here I feel like that's an unnecessary hurdle and is testable as is